regalia-moshi/build.gradle

```diff
@@ -25,15 +27,16 @@ android {
 }
 
 dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
 
-    api 'com.squareup.moshi:moshi:1.5.0'
+    api 'com.squareup.moshi:moshi:1.9.2'
+    api 'com.squareup.moshi:moshi-kotlin:1.9.2'
```

Comment:
We shouldn't force this dependency. What if they want to use kotlin with codegen instead?

Reply:
You're quite right friend, I shouldn't have made this pull request since it's suited for personal usage.

Reply:
I appreciate the PR and think it can be merged just fine with this repo, with some modifications made.